Bot/TynibotHost.cs

```diff
@@ -50,7 +50,8 @@ public TynibotHost()
 
             Client = new DiscordSocketClient(new DiscordSocketConfig()
             {
-                LogLevel = LogSeverity.Debug
+                LogLevel = LogSeverity.Debug,
+                UseInteractionSnowflakeDate = false
```

Comment:
Moving to not using interaction snowflake date - discord has had a couple time skew issues recently, these make the request fail the internal 3s timeout check even though the response is still good. Choosing to not use snowflake date means that we use local system time instead.